src/nvim/tui/tui.c

```diff
@@ -913,12 +932,15 @@ static void out(void *ctx, const char *str, size_t len)
   data->bufpos += len;
 }
 
-static void unibi_set_if_empty(unibi_term *ut, enum unibi_string str,
+static bool unibi_set_if_empty(unibi_term *ut, enum unibi_string str,
     const char *val)
 {
-  if (!unibi_get_str(ut, str)) {
+  const char *temp = unibi_get_str(ut, str);
```

Comment:
Leftover of a debug session. Removed the temp variable. Kept returning the bool as it helps debug and should be optimized out by compiler anyway.